.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prettier.configPath": ".prettierrc",
+  "editor.formatOnSave": true
+}
```

Comment:
If I go and change my workspace settings, will it show up as a change to this file? I think it would?

Reply:
oops i meant to just add extensions.json in this PR! we might want to do this too but I meant to separate it out so we can discuss it separately